test_libs/pyspec/eth2spec/test/epoch_processing/test_process_crosslinks.py

```diff
@@ -103,19 +104,24 @@ def test_single_crosslink_update_from_previous_epoch(state):
 
 @spec_state_test
 def test_double_late_crosslink(state):
+    if spec.get_epoch_committee_count(state, spec.get_current_epoch(state)) < spec.SHARD_COUNT:
+        print("warning: ignoring test, test-assumptions are incompatible with configuration")
+        return
+
     next_epoch(state)
     state.slot += 4
 
     attestation_1 = get_valid_attestation(state, signed=True)
     fill_aggregate_attestation(state, attestation_1)
 
-    # add attestation_1 in the next epoch
+    # add attestation_1 to next epoch
     next_epoch(state)
     add_attestation_to_state(state, attestation_1, state.slot + 1)
 
     for slot in range(spec.SLOTS_PER_EPOCH):
-        attestation_2 = get_valid_attestation(state, signed=True)
+        attestation_2 = get_valid_attestation(state, signed=False)
```

Comment:
Follow up with my previous comment: https://github.com/ethereum/eth2.0-specs/pull/1102#issuecomment-494232533
I just figured out you were replying to me in the third paragraph? https://github.com/ethereum/eth2.0-specs/pull/1102#issuecomment-494522605 

I think I failed to explain what I meant! 😅  Sorry, I meant making it more explicit! 

Either we can
1. have `def get_valid_attestation(state, signed, slot=None)`, so it's restricted to set signed *every time* you call the function; or
2. have `def get_valid_attestation(state, slot=None, signed=False)`, and make sure it's really clear that "default value is `False`". In that case, IMO having both `get_valid_attestation(state, signed=False)` and `get_valid_attestation(state)` exist are confusing. We should never set `signed=False` when calling it.

Reply:
Option 2 then.

Reply:
See 
b12031b